var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class LlvmAmdgpu(CMakePackage):
+    """Toolkit for the construction of highly optimized compilers,
+       optimizers, and run-time environments."""
+
+    homepage = "https://github.com/RadeonOpenCompute/llvm-project"
+    url      = "https://github.com/RadeonOpenCompute/llvm-project/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='4878fa85473b24d88edcc89938441edc85d2e8a785e567b7bd7ce274ecc2fd9c')
+
+    variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+
+    depends_on('cmake@3.5.2', type='build')
+
```

Comment:
```suggestion
depends_on('python', type='build')
```

Reply:
@haampie , Will look into it..I was able to build it with out explicitly adding it .Is there a specific need to include it.. Appreciate your inputs.

Reply:
If the package requires Python at build time, you should always add an explicit dependency, even if you happen to have Python installed on your OS.

Reply:
I have just added it because AUR seems to do it as well: https://aur.archlinux.org/packages/llvm-amdgpu/ and the llvm package seems to list it as a build dep as well